susemanager-utils/susemanager-sls/salt/bootstrap/susemanager.conf

```diff
@@ -12,3 +12,5 @@ start_event_grains:
   - machine_id
   - saltboot_initrd
   - susemanager
+use_superseded:
+  - module.run
```

Comment:
As it should now work with both, old and new syntax we should not enforce it.
A customer who has own written state files and did not port it yet to use the compat module will hard fail for new clients.
And as this config file is managed by SUSE Manager he cannot change it.

Reply:
We should do this in the testsuite, so we can be sure that it works.

Reply:
That's true. We should not enable this by default.